src/quicr_client_raw_session.cpp

```diff
@@ -174,7 +205,9 @@ QuicRClientRawSession::make_transport(RelayInfo& relay_info,
     throw std::runtime_error(log_msg.str());
   }
 
-  transport_stream_id = transport->createStream(transport_context_id, false);
+  _stream_mode = stream_mode;
+  _control_stream_id =
+    transport->createStream(transport_context_id, is_reliable());
```

Comment:
I think this needs one previous call to createStream with unreliable, but may not need to save off the datagram stream.